### PR TITLE
add kernel version helpers

### PR DESCRIPTION
**Pull request by roikol**

### CODE REVIEW

File: helpers/kernel_version.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	var buf [65]byte
+	for i, b := range uname.Release {
+		buf[i] = byte(b)
+	}
+	ver := string(buf[:])
+	ver = strings.Trim(ver, "\x00")
+	return ver
+}
+
+func kernelIsAtLeast(verMajor, verMinor int) (bool, error) {
```
Comment (rafaeldtinoco) [inline helpers/kernel_version.go:25]: Im merging today a change that will be more complete (generic usage) than this one (simple just for your needs). Is it possible for you to wait to use the OSInfo API ?

Comment (rafaeldtinoco) [reply helpers/kernel_version.go:25]: image

Comment (rafaeldtinoco) [reply helpers/kernel_version.go:25]: You will be able to run helpers.NewOSInfoRelease() to get the kernel version only (without /etc/os-release) and use its internal functions (comparing kernel versions etc).

Comment (rafaeldtinoco) [reply helpers/kernel_version.go:25]:
```
$ sudo ./dist/tracee-ebpf --debug --trace uid=1000 --trace pid=new --trace event=execve --trace event=cap_capable --output option:detect-syscall
OSInfo: PRETTY_NAME: "Ubuntu 21.04"
OSInfo: VERSION_ID: "21.04"
OSInfo: VERSION_CODENAME: hirsute
OSInfo: KERNEL_RELEASE: 5.11.0-31-generic
OSInfo: VERSION: "21.04 (Hirsute Hippo)"
OSInfo: ID: ubuntu
OSInfo: ID_LIKE: debian
TEST: ubuntu
TEST: 5.11.0-31-generic
BTF: bpfenv = false, btfenv = false, vmlinux = true
BPF: using embedded BPF object
unpacked CO:RE bpf object file into memory
```

Comment (rafaeldtinoco) [reply helpers/kernel_version.go:25]: Could you please review https://github.com/aquasecurity/libbpfgo/pull/67 and see if that would satisfy your needs on this one ?

Comment (rafaeldtinoco) [reply helpers/kernel_version.go:25]: You may use either:

`OSInfo.CompareOSBaseKernelRelease("5.11.0"))`

or just the function

`CompareOSBaseKernelRelease(baseVersion, givenVersion)`

from the API.

Comment (roikol) [reply helpers/kernel_version.go:25]: yes it looks very good and should address all my use cases, thank you for that

**Comment by roikol** [comment]
i'm closing this PR due to #67

**Pull request closed without merging**